sale_channel_mirakl/tests/common.py

```diff
@@ -0,0 +1,577 @@
+                        "phone_secondary": "0688430945",
+                        "state": "",
+                        "street_1": "21 avenue paul langevin ",
+                        "street_2": "",
+                        "zip_code": "92260",
+                    },
+                    "civility": "",
+                    "customer_id": "3786926",
+                    "firstname": "maria",
+                    "lastname": "nicolo",
+                    "locale": "",
+                    "shipping_address": {
+                        "additional_info": "",
+                        "city": "fontenay-aux-roses",
+                        "civility": "Mme",
+                        "company": "",
+                        "country": "France",
+                        "country_iso_code": None,
+                        "firstname": "Albert",
+                        "lastname": "Einstein",
+                        "phone": "0688430945",
+                        "phone_secondary": "0688430945",
+                        "state": "",
+                        "street_1": "21 avenue paul langevin ",
+                        "street_2": "",
+                        "zip_code": "92260",
+                    },
+                },
+                "customer_debited_date": "2016-11-18T" "15:31:03.190Z",
+                "has_customer_message": "",
+                "has_incident": "",
+                "has_invoice": "",
+                "last_updated_date": "2017-02-21T23:00:01Z",
+                "leadtime_to_ship": 0,
+                "order_additional_fields": [],
+                "order_id": "1611DBLLZ-MP002191-A",
+                "order_lines": [
+                    {
+                        "can_refund": "",
+                        "cancelations": [],
+                        "category_code": "JJ01-003-002",
+                        "category_label": "FIGURINES",
+                        "commission_fee": 11.25,
+                        "commission_rate_vat": 0,
+                        "commission_taxes": [
+                            {"amount": 0, "code": "TAXZERO", "rate": 0}
+                        ],
+                        "commission_vat": 0,
+                        "created_date": "2016-11-18T13:31:55Z",
+                        "debited_date": "2016-11-18T15:31:03Z",
+                        "description": "Avec ses quatre étages "
+                        "de parking,"
+                        " ses pompes à "
+                        "essence et un "
+                        "marquage d'héliport, les "
+                        "conducteurs impatients "
+                        "vont et "
+                        "viennent dans toutes les "
+                        "directions.INFORMATIONS "
+                        "TECHNIQUES :  - Age : 3A+",
+                        "last_updated_date": "2017-02-21T23:00:01Z",
+                        "offer_id": 41601,
+                        "offer_sku": "29182572",
```

Comment:
You probably have to fill the content with demo data.
If something else (another module) change a `default_code` (or other), you test could fail.